examples/label_molecule/label_molecule.py

```diff
@@ -1,18 +1,16 @@
-#!/bin/env python
+#!/usr/bin/env python
 
+from openforcefield.topology import Molecule, Topology
+from openforcefield.typing.engines.smirnoff import ForceField
 
-from openforcefield.utils import *
-from openforcefield.typing.engines.smirnoff import get_molecule_parameterIDs, ForceField
-
-# Create an oemol
-mol = OEMol()
-OEParseSmiles(mol, 'CCC')
-OEAddExplicitHydrogens(mol)
-
+# Create a propane molecule and its topology.
+molecule = Molecule.from_smiles('CCC')
```

Comment:
> We could expose the toolkit_registry parameter in an "advanced" example which shows more customization. What do you think?

I think this is a good idea. In general, I think the approach "use anything that works" is what is emphasized by the API so it might be good for an introductory example, and I agree that an example clarifying what's going on under the hood should be useful for the more advanced user.